.travis.yml

```diff
@@ -28,6 +25,7 @@ services:
 
 before_install:
 - java -version
+- python -V
```

Comment:
you don't need this, travis already outputs python version anyway: https://travis-ci.org/aio-libs/aiocassandra/jobs/340881258#L538